yaksh/bash_code_evaluator.py

```diff
@@ -9,26 +9,31 @@
 import importlib
 
 # local imports
-from .code_evaluator import CodeEvaluator
+from .base_evaluator import BaseEvaluator
 from .file_utils import copy_files, delete_files
 
 
-class BashCodeEvaluator(CodeEvaluator):
+class BashCodeEvaluator(BaseEvaluator):
     # Private Protocol ##########
-    def setup(self):
-        super(BashCodeEvaluator, self).setup()
+    def __init__(self, metadata, test_case_data):
```

Comment:
In the next refactor you could actually refactor this to use all keyword args like `__init__(self, user_answer, file_paths, ...)` and use **metadata when instantiating it?